code/modules/client/preference_setup/loadout/lists/suits.dm

```diff
@@ -170,7 +170,7 @@
 /datum/gear/suit/akira
 	display_name = "red jacket"
 	path = /obj/item/clothing/suit/storage/akira
-
+/*
 /datum/gear/suit/radhat
```

Comment:
This ruins a few peoples character slots

Reply:
Ah. Is there a cleaner way to deal with that than removing it completely?

Reply:
we are testing rn to make sure the damage isn't too terrible